jobs/async-upload/requirements.txt

```diff
@@ -16,8 +17,11 @@ eval-type-backport==0.2.2 ; python_version >= "3.11" and python_version < "4.0"
 filelock==3.18.0 ; python_version >= "3.11" and python_version < "4.0"
 frozenlist==1.7.0 ; python_version >= "3.11" and python_version < "4.0"
 fsspec==2025.7.0 ; python_version >= "3.11" and python_version < "4.0"
-hf-xet==1.1.5 ; python_version >= "3.11" and python_version < "4.0" and (platform_machine == "x86_64" or platform_machine == "amd64" or platform_machine == "arm64" or platform_machine == "aarch64")
-huggingface-hub==0.35.3 ; python_version >= "3.11" and python_version < "4.0"
+h11==0.16.0 ; python_version >= "3.11" and python_version < "4.0"
+hf-xet==1.2.0 ; python_version >= "3.11" and python_version < "4.0" and (platform_machine == "x86_64" or platform_machine == "amd64" or platform_machine == "AMD64" or platform_machine == "arm64" or platform_machine == "aarch64")
+httpcore==1.0.9 ; python_version >= "3.11" and python_version < "4.0"
+httpx==0.28.1 ; python_version >= "3.11" and python_version < "4.0"
+huggingface-hub==1.0.1 ; python_version >= "3.11" and python_version < "4.0"
```

Comment:
bump also manually the requirements.txt following the HF library bump, as also originating from the MR py client

a mayor version bump

with this manual sync, 
/lgtm
/approve